packages/internal-test-helpers/lib/test-resolver.js

```diff
@@ -0,0 +1,43 @@
+import { compile } from 'ember-template-compiler';
+
+class Resolver {
+  constructor() {
+    this._registered = {};
+    this.constructor.lastInstance = this;
```

Comment:
The instance is created by `Application`. We could access it via `application.__registry__.resolver` however that would couple this to the arrangement of "registry has a resolver", which isn't true for glimmer/di. Of course we will have a FF for that anyway so we could change the logic. It just feels ugly to be relying on `reaching.deep.into.__private__.stuff` to find the instance.